app/src/main/AndroidManifest.xml

```diff
@@ -1,27 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="org.openobservatory.ooniprobe">
 
+    <uses-sdk tools:overrideLibrary="us.feras.mdv" />
```

Comment:
Not something I wrote, I guess is something from https://github.com/falnatsheh/MarkdownView